client/my-sites/plugins/search-box-header/index.jsx

```diff
@@ -15,21 +15,22 @@ const SearchBox = ( { isMobile, doSearch, searchTerm, searchBoxRef } ) => {
 	return (
 		<div className="search-box-header__searchbox">
 			<Search
+				ref={ searchBoxRef }
 				pinned={ isMobile }
 				fitsContainer={ isMobile }
 				onSearch={ doSearch }
 				defaultValue={ searchTerm }
 				placeholder={ translate( 'Try searching "ecommerce"' ) }
 				delaySearch={ true }
 				recordEvent={ recordSearchEvent }
-				ref={ searchBoxRef }
+				searchMode="on-enter"
```

Comment:
Can we move this change to another PR? I'm a little wary of assuming everyone knows enter = submit ([we hide the submit button](https://github.com/Automattic/wp-calypso/issues/62540)) .

Reply:
@lsl I think this is actually not a submit button, it seems to be a button used to open the search bar on mobile

Reply:
but divided this one into two PRs for testing reasons